.github/workflows/continuous.yml

```diff
@@ -0,0 +1,91 @@
+# ------------------------------------------------------------------------------
+# <auto-generated>
+#
+#     This code was generated.
+#
+#     - To turn off auto-generation set:
+#
+#         [GitHubActions (AutoGenerate = false)]
+#
+#     - To trigger manual generation invoke:
+#
+#         nuke --generate-configuration GitHubActions_continuous --host GitHubActions
+#
+# </auto-generated>
+# ------------------------------------------------------------------------------
+
+name: continuous
+
+on: [push]
+
+jobs:
+  macOS-10_14:
```

Comment:
you can use a matrix here, as each of the different steps are the same.

eg
```
jobs:
  build:
    strategy:
      matrix: os: [macOS-10.14, ubuntu-16.04, ...]
    name: Build ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
```

Reply:
Still tuning :) thanks for the tip